frontend/app/component/error/Error.module.css

```diff
@@ -0,0 +1,38 @@
+.errorContainer {
```

Comment:
I think the consensus will be to use kebab/dash case for classnames. 
anyway, nothing case sensitive :)

Reply:
A name with a dash is not usable as a property of the css module import without using string quotes, but I can change this to an single name since it is scoped to this component :+1: